src/envoy/utils/filter_state_utils.cc

```diff
@@ -44,6 +53,24 @@ absl::string_view getStringFilterState(
   return filter_state.getDataReadOnly<StringAccessor>(data_name).asString();
 }
 
+void setErrorFilterState(Envoy::StreamInfo::FilterState& filter_state,
+                         const google::rpc::Status& status) {
+  auto state = std::make_unique<RpcStatusWrapper>();
+  state->status_ = status;
+  filter_state.setData(kErrorRpcStatus, std::move(state),
+                       Envoy::StreamInfo::FilterState::StateType::ReadOnly);
+}
+
+bool hasErrorFilterState(Envoy::StreamInfo::FilterState& filter_state) {
+  return filter_state.hasData<RpcStatusWrapper>(kErrorRpcStatus);
+}
+
+google::rpc::Status getErrorFilterState(
```

Comment:
not to make a copy in "get" function,   and return "const &"

if caller wants to make a copy,  it can copy it on its own.

Reply:
Done